src/content/blog/2024-09-29-fighting-the-invisible-Battle-my-story-of-health-struggles.mdx

```diff
@@ -0,0 +1,170 @@
+
+> What happened 3 years ago as I moved to Germany, and I was away from my family for 10 months, which caused me mental stress. With no support in a new country, it started impacting my health.
+
+### Impact on work
+
+After moving to Germany, I started underperforming at my first job, and I saw no future at the organization. I was lucky to join Celonis in Aug 2022, where my co-workers and employer gave me the support I wanted. 
+
+For the first time in eight months, someone believed in me. I met some friends from the Indian community, which gave me some confidence back. My family joined me soon, but the damage was already done.
+
+But I kept questioning myself if I was good enough.
+
+## Questioning myself on every project
+
+At Celonis, there were and still are many people who believed in me, but I want to be honest: I did not; I kept questioning myself on every project I worked on.
+
+I used to feel bad when I received PR review comments or broke an app due to a coding mistake I made.
+
+<BlogImage src={doubt} alt="Doubting myself" />
+
+
+Photo by <a href="https://unsplash.com/@dargonesti?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Ludovic Migneault</a> on <a href="https://unsplash.com/photos/man-in-gray-button-up-shirt-B9YbNbaemMI?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+  
+I was working on some initiatives, which were multi-quarter efforts, and I used to wonder why it was taking this long and why I was doing this. My managers and team believed I was the best person to do this and would finish this project. 
+
+
+### Making my health worse
+
+One thing I love more than anything is getting myself involved in the community; after joining Celonis, I started speaking at conferences worldwide and focusing less on my health. From October 2022 to October 2024, I was in more than 20 countries, sharing my knowledge on Angular, Nx, GitHub, and Open Source. 
+
+And I could feel physically and mentally I was finding it hard; I was sleeping for more than 12 hours a day, still trying to find time to do public speaking.
+
+I spoke at conferences every month from October 2022 to November 2023, requiring significant effort to prepare content, practice my talk, and travel. This constant cycle was physically and mentally exhausting.
+
+Forcing myself was a bad call; always listen to your body. 
+
+<BlogImage src={wellness} alt="If you do not make time for your wellness, you will be forced to make time for your illness" />
+
+
+> 'If you do not make time for your wellness, you will be forced to make time for your illness' #readthatagain
+
+The above lines are so true. My body started forcing me to take that time, but I didn't like it.
+
+Once I was diagnosed, I realized that going through mental and physical stress was the reason for diabetes.
```

Comment:
The article is an inspiring account of personal growth, but it would benefit from more reflection on key moments. For example, when you mention being diagnosed with diabetes, more detail about how that news affected you in the moment and what the hardest part of the journey was would resonate with the reader on a deeper emotional level. For example you mentioned somewhere that you gave up sugar. How did you do that? It's very difficult to give up sugar.